.github/PULL_REQUEST_TEMPLATE

```diff
@@ -1,10 +1,31 @@
-## What changes were proposed in this pull request?
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+1. If this is your first time, please read our contributor guidelines: https://spark.apache.org/contributing.html
+2. Ensure you have added or ran the appropriate tests for your PR: https://spark.apache.org/developer-tools.html
+3. If the PR is unfinished, add `[WIP]` in your PR title.
```

Comment:
In this line, can we explicitly guide more about the PR title, too? Something like the following. (I borrowed this from the contributor guideline because many people missed this.)
> The PR title should be of the form [SPARK-xxxx][COMPONENT] Title, where SPARK-xxxx is the relevant JIRA number

Reply:
Oh. It will be the same comment with @maropu 's comment.